package.json

```diff
@@ -19,7 +19,13 @@
     "bump:version:major-prerelease": "node ./scripts/deploy/manual-major-prerelease.mjs",
     "promote:npm:beta": "nx run-many --target=promote:npm:beta",
     "promote:npm:latest": "nx run-many --target=promote:npm:latest",
-    "notify:docs": "node ./scripts/notify-docs/published-ui-kit.mjs"
+    "notify:docs": "node ./scripts/notify-docs/published-ui-kit.mjs",
+    "release": "npm run nx:graph && npm run release:phase0 && npm run release:phase1 && npm run release:phase2",
+    "nx:graph": "nx graph --file=topology.json",
+    "release:install-npm-proxy": "npx -p=@coveo/release install-npm-proxy",
+    "release:phase0": "exit 38",
```

Comment:
Temporary.

Number corresponds with `ENOSYS 38 Function not implemented` in case you were wondering.